### PR TITLE
archive(8.2) exclusion: logically remove versions 8.2/3.10.0. Reviewable steps.

### PR DESCRIPTION
**Pull request by pepopowitz**
## Description

Part of #4527.

Includes the smaller items for removing versions 8.2/3.10.0 from the main docs, to keep the PR in a reviewable state.

This means removing the versions _logically_, but not yet physically. When this PR is merged, the versions will be effectively archived. An upcoming PR will remove the version physically. 

This PR also includes some more significant changes to the htaccess file. 

## Included in this PR.

- Remove v8.2 & Optimize v3.10.0 from the version manifest files, so that they're not included in the deployed site
- Redirect all 8.2 / 3.10.0  traffic to `unsupported.docs.camunda.io/8.2` (or `stage.unsupported.docs.camunda.io/8.2` for the staging build)
- Move "8.2 / 3.10.0" versions to the "Unsupported versions" sub-list in the top nav version selector, redirecting to `unsupported.docs.camunda.io8.2`. 
- Remove v8.2 & Optimize v3.10.0 from the sitemap ignore patterns.
- More significant redirect rule changes: 
   1. Document structure/organization of .htaccess file.
   2. Reorganize a few things to align with that documentation.
   3. Remove all redirects that were introduced for version 8.2 or earlier. This is more than what I usually do, which is to remove redirects that _specify_ 8.2. If we are no longer supporting versions 8.2 or earlier, I think we can stop supporting redirect rules for those versions. This allows us to clean up this file on a regular basis and keep it from becoming massive.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

- [x] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process).

### CODE REVIEW

File: static/.htaccess [hunk 1]
```diff
@@ -1,3 +1,31 @@
+# Welcome to the redirect party! 
+#  ____  ____  ____  __  ____  ____  ___  ____    ____   __   ____  ____  _  _  _   
+# (  _ \(  __)(    \(  )(  _ \(  __)/ __)(_  _)  (  _ \ / _\ (  _ \(_  _)( \/ )/ \  
+#  )   / ) _)  ) D ( )(  )   / ) _)( (__   )(     ) __//    \ )   /  )(   )  / \_/  
+# (__\_)(____)(____/(__)(__\_)(____)\___) (__)   (__)  \_/\_/(__\_) (__) (__/  (_)  
+#
```
Comment (pepopowitz) [inline static/.htaccess:6]: oof, this looks bad in the diffed version. Standalone it is a lot more fun. I can remove it if you want.

File: static/.htaccess [hunk 2]
```diff
@@ -6,13 +34,45 @@ RewriteRule (.*) https://%{SERVER_NAME}/$1 [R=301,L]
...
+RewriteRule ^docs/1.1/(.*)$ https://unsupported.docs.camunda.io/1.1/docs/$1 [R=301,L]
+RewriteRule ^docs/1.2/(.*)$ https://unsupported.docs.camunda.io/1.2/docs/$1 [R=301,L]
+RewriteRule ^docs/1.3/(.*)$ https://unsupported.docs.camunda.io/1.3/docs/$1 [R=301,L]
+RewriteRule ^optimize/3.7.0/(.*)$ https://unsupported.docs.camunda.io/1.3/optimize/$1 [R=301,L]
+RewriteRule ^docs/8.0/(.*)$ https://unsupported.docs.camunda.io/8.0/docs/$1 [R=301,L]
+RewriteRule ^optimize/3.8.0/(.*)$ https://unsupported.docs.camunda.io/8.0/optimize/$1 [R=301,L]
+RewriteRule ^docs/8.1/(.*)$ https://unsupported.docs.camunda.io/8.1/docs/$1 [R=301,L]
+RewriteRule ^optimize/3.9.0/(.*)$ https://unsupported.docs.camunda.io/8.1/optimize/$1 [R=301,L]
+RewriteRule ^docs/8.2/(.*)$ https://unsupported.docs.camunda.io/8.2/docs/$1 [R=301,L]
+RewriteRule ^optimize/3.10.0/(.*)$ https://unsupported.docs.camunda.io/8.2/optimize/$1 [R=301,L]
```
Comment (pepopowitz) [inline static/.htaccess:73]: This entire block was moved from the bottom of the file, but these two lines specifically are added to redirect the newly unsupported 8.2 version.

File: static/.htaccess [hunk 3]
```diff
@@ -418,469 +430,4 @@ RewriteRule ^docs/components/modeler/web-modeler/save-and-deploy/?$ /docs/compon
 RewriteRule ^docs/next/components/modeler/web-modeler/start-instance/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/ [R=301,L]
 RewriteRule ^docs/next/components/modeler/web-modeler/save-and-deploy/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/#deploy-a-process [R=301,L,NE]
 
-# Modeler start instance and deploy page renaming and merging 8.2
```
Comment (pepopowitz) [inline static/.htaccess:421]: There's a lot of removals from this point forward. All of these rules at the bottom were introduced prior to 8.3 shipping, which I think means that they were part of 8.2, and can be sunsetted with that version.

There are two rules which aren't versioned, which I moved to above sections. (One for the blog, one for slashes at the end of assets.)

File: static/.htaccess (hunk 1 above)
Comment (akeller) [reply static/.htaccess:6]: Keep it! We should have a little fun every now and then.

**Review by akeller — DISMISSED** [review]
🚀

**Comment by pepopowitz** [comment]
ok @akeller I finally got this build green. I had to revive a couple rules I'd deleted, to support product links.....and then I ran into a build caching issue, which I'll clarify in #ask-documentation in case anyone else runs into it.

When you get a chance, a re-review would be grand!

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
> ok @akeller I finally got this build green. I had to revive a couple rules I'd deleted, to support product links.....and then I ran into a build caching issue, which I'll clarify in #ask-documentation in case anyone else runs into it.
> 
> When you get a chance, a re-review would be grand!
> 
>  [ 68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6252566e34525764696155373441504178552f67697068792e676966 ](https://camo.githubusercontent.com/21b745ec5482bfc8d2428a88255ffb7c3a77411bd354da725b83fbb66901d74e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6252566e34525764696155373441504178552f67697068792e676966) [ ](https://camo.githubusercontent.com/21b745ec5482bfc8d2428a88255ffb7c3a77411bd354da725b83fbb66901d74e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6252566e34525764696155373441504178552f67697068792e676966)

I reviewed your commits, all good!

**Pull request merged**